internal/commands/build/build.go

```diff
@@ -45,20 +45,21 @@ func Cmd(dockerCli command.Cli) *cobra.Command {
 	var opts buildOptions
 	cmd := &cobra.Command{
 		Use:     "build [OPTIONS] [CONTEXT_PATH]",
-		Short:   "Build service images for the application",
-		Example: `$ docker app build --tag my/app:1.0.0 .`,
+		Short:   "Build an App Image from a .dockerapp App Definition",
+		Example: `$ docker app build .
+$ docker app build . -f myapp.dockerapp -t myrepo/myapp:1.0.0`,
```

Comment:
command is `[OPTIONS] [CONTEXT_PATH]` so I think `.` should be at the end in the example

```suggestion
$ docker app build -f myapp.dockerapp -t myrepo/myapp:1.0.0` .,
```

Reply:
this also would be consistent with docker build UX